.travis.yml

```diff
@@ -96,7 +98,8 @@ before_install:
     if [[ "$UNSUPPORTED" == true ]]; then
       pip install 3to2
       export PIP_NO_COMPILE=1
-      sed -E -i.bak '/^(coala|mypy-lang)/d' requirements.txt bear-requirements.txt
+      sed -i.bak '/^coala/d' requirements.txt
+      sed -i.bak '/^mypy-lang/d' bear-requirements.txt
```

Comment:
In your latest commit (a2b0f99) , you also remove this line which appeared in b73aefdab (which this review discussion is about)

> sed -i.bak '/^coala/d' requirements.txt

That line is necessary.  My prior review comment here did not say to remove that line.

There are comments in `.travis.yml` explaining why `coala` and `mypy-lang` need special treatment on Python 2.7 & 3.3 .  Your task is *only* to shorten the lines - not change functionality.  The existing `sed` can be rewritten may ways to achieve the same functionality.